### DISCUSSION TITLE
Not falling back to SPA route if prerender enabled

### DISCUSSION THREAD

**Discussion by dominiczy** [vikejs/vike#563] [Discussion]
I want to enable SSR/SSG for some routes of my react app.

The SSR-enabled routes are working, but when I visit a non-SSR (eg. SPA-only) route, I get a 404, instead of it falling back to the client side react spa route.

In `vite dev` it works fine but when building and serving I get 404

Any ideas?

Possibly related to this one https://github.com/brillout/vite-plugin-ssr/discussions/558 but no answers there so far..

**Comment by brillout** [comment]
Interesting use case. I think a catch-all route should do the trick.

**Reply by dominiczy** [reply]
Where would I set up the catch-all route?

**Reply by dominiczy** [reply]
I've created an `all.page.server.jsx` but that doesn't fix it

**Comment by dominiczy** [comment]
Easy way to reproduce the issue: add `{ prerender: true }` or even `{ prerender: { partial: true } }` to the [react-router example](https://github.com/brillout/vite-plugin-ssr/tree/main/examples/react-router).
So it seems to be a prerender issue?
